api-reference/upload-file-api/client-side-file-upload.md

```diff
@@ -106,9 +106,13 @@ The JSON-encoded response contains details of the file which can have the follow
 | url               | A publicly accessible URL of the file.                                                                                                                                                                                                  |
 | thumbnailUrl      | In the case of an image, a small thumbnail URL.                                                                                                                                                                                         |
 | fileType          | The type of file could be either `image` or `non-image`.                                                                                                                                                                                |
-| height            | Height of the image in pixels (Only for images)                                                                                                                                                                                         |
-| width             | Width of the image in pixels (Only for Images)                                                                                                                                                                                          |
+| height            | Height of the media file in pixels                                                                                                                                                                                                      |
+| width             | Width of the media file in pixels                                                                                                                                                                                                       |
 | size              | Size of the file in Bytes                                                                                                                                                                                                               |
+| bit_rate          | Bitrate of the media file                                                                                                                                                                                                               |
+| video_codec       | Primary video codec for the media file                                                                                                                                                                                                  |
+| audio_codec       | Primary audio codec for the media file                                                                                                                                                                                                  |
```

Comment:
What is primary audio codec? should we say first stream instead?  Or better add a note saying that all video related attributes are for first stream only.

Reply:
First stream makes more sense. "Primary" is usually the default or most-used codec for a file but that isn't really what we're finding here.

Reply:
This is fixed.